build_library/grub_install.sh

```diff
@@ -143,6 +143,13 @@ case "${FLAGS_target}" in
         sudo mkdir -p "${ESP_DIR}/EFI/boot"
         sudo cp "${ESP_DIR}/${GRUB_DIR}/${CORE_NAME}" \
             "${ESP_DIR}/EFI/boot/bootx64.efi"
+	# Use the test keys for signing unofficial builds
+	if [[ ${COREOS_OFFICIAL:-0} -ne 1 ]]; then
+            sudo sbsign --key /usr/share/sb_keys/DB.key \
```

Comment:
Perhaps sign `"${ESP_DIR}/${GRUB_DIR}/${CORE_NAME}"` instead (it expands to `"${ESP_DIR}/coreos/grub/x86_64-efi/core.efi"`) Then in that directory there is both a core.efi and a core.efi.signed which may be a useful reference to someone digging around trying to understand the boot process. It is less obvious when it is core.efi that is unsigned but bootx64.efi which lives elsewhere that is signed.